SpacetimeDB.ClientSDK.csproj

```diff
@@ -5,7 +5,6 @@
     <LangVersion>9</LangVersion>
     <ImplicitUsings>disable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
```

Comment:
Is this removed just because we don't need it for tests and such?

Reply:
it prevented `dotnet pack` from working properly and this fixed it, I have no context on what it does :shrug: 
https://steveellwoodnlc.medium.com/error-nu5026-the-file-to-be-packed-was-not-found-on-disk-18bfbc6be4a

Reply:
Weird. This line basically integrates packing right into `dotnet build` so that you don't have to run `dotnet pack` separately... but I had no idea it might conflict with the latter.

Removing sgtm.

Reply:
oh! then I can just update the release docs to say run `dotnet build --configuration Release` instead. looks like it works for me!

Reply:
Noo, I was actually fine with removing that line for consistency with other repos.

Also I suspect it's somewhat inefficient now that we have tests - because it will probably build the NuGet package even for `dotnet test`, which is kinda wasteful.

Reply:
restored!